design-library/src/components/BccInput/BccInput.css

```
@@ -1,6 +1,6 @@
 @layer components {
     .bcc-input {
-        @apply w-full flex items-center min-h-[2.5rem] px-4 py-2 text-body-sm rounded-lg text-primary bg-primary border border-on-primary focus:outline-1 focus:outline-emphasis;
+        @apply w-full flex items-center min-h-[2.5rem] px-4 py-2 text-body rounded-lg text-primary bg-primary border border-on-primary focus:outline-1 focus:outline-emphasis;
```

I would suggest doing the same thing as [Tailwind UI](https://tailwindui.com/components/application-ui/forms/input-groups) does and still make it smaller on larger screens:

```suggestion
        @apply w-full flex items-center min-h-[2.5rem] px-4 py-2 text-body sm:text-body-sm rounded-lg text-primary bg-primary border border-on-primary focus:outline-1 focus:outline-emphasis;
```